llvm/lib/Bitcode/Writer/BitcodeWriter.cpp

```diff
@@ -23,6 +23,7 @@
 #include "llvm/ADT/SmallVector.h"
 #include "llvm/ADT/StringMap.h"
 #include "llvm/ADT/StringRef.h"
+#include "llvm/Analysis/ModuleSummaryAnalysis.h"
```

Comment:
I'm not sure including an additional header is superior to just including the extern here below - why make this change?

Reply:
Thanks for the background. I'm not sure how I feel about going through and changing all of these as I think it will cause a lot of churn and add compile time. Let's wait to see if the coding standard change is accepted.